src/xcode/ENA/ENA/Source/Extensions/TimeInterval+Minutes.swift

```diff
@@ -0,0 +1,17 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+
+extension TimeInterval {
+
+	init?(minutes: Int?) {
```

Comment:
why not?
```suggestion
	init?(minutes: Int) {
```

Reply:
Because this is for my very specific use case where I need it to return nil if minutes is nil 😅

Reply:
Ah okay :D